platforms/access-bridge/src/pipeline/middleware.ts

```diff
@@ -59,6 +59,7 @@ export class Middleware {
     if (err instanceof Stripe.errors.StripeError) {
       const accessBridgeError = handleStripeError(err);
       sendErrors(res, accessBridgeError);
+      return;
```

Comment:
Do wee need these empty returns?

Reply:
This is pretty much the case with javascript and most of the languages 🙂 
```
function test() {
  // do something
  return; // This exits the `test` function
}

function useTest() {
  if (something) {
    test(); // Call the `test` function
    // If you want to prevent further logic from running, you'd return here.
  }
  
  // This "other logic" will execute if the return isn't present after calling `test()`
  // other logic
}
```

Reply:
@AntonLantukh Do you have any concerns, or is this just an informative question? If it's the latter, we should discuss it in our shared Slack channel, not here.

Reply:
@kiremitrov123 my comment was mostly about further usage of `return` for `notFoundErrorHandler` and `globalErrorHandler` where the execution of the function ends with `return` and no other conditions are expected. The example with `err instanceof Stripe.errors.StripeError` condition makes sense to me.

If you set `return` for `notFoundErrorHandler` will the execution go further to `globalErrorHandler`? Should we check the error type in  `notFoundErrorHandler` and call `next` if this is not related to api?

https://expressjs.com/en/guide/error-handling.html#error-handling

Reply:
@darkoatanasovski I looking into the problem we tried to solve here. Fix for tests looks good to me.

Reply:
@AntonLantukh the question was addressed exactly on top of the Stripe one and I assumed you asked for every one of them. For the last return and one on the `notFoundErrorHandler`, they can be removed as it doesn't make any difference.